nessie/sql_templates/create_rds_indexes.template.sql

```diff
@@ -158,7 +158,17 @@ CREATE TABLE IF NOT EXISTS {rds_schema_student}.ethnicities
 CREATE INDEX IF NOT EXISTS students_ethnicities_sid_idx ON {rds_schema_student}.ethnicities (sid);
 CREATE INDEX IF NOT EXISTS students_ethnicities_sid_idx ON {rds_schema_student}.ethnicities (ethnicity);
 
-CREATE TABLE IF NOT EXISTS {redshift_schema_student}.student_holds
```

Comment:
This `redshift_` prefix is an unrelated bug that's been hanging around for a while.